From bgat@billgatliff.com  Fri Dec 18 11:57:58 2009
From: Bill Gatliff <bgat@billgatliff.com>
Date: Fri, 18 Dec 2009 09:57:22 -0600
Subject: USB: Fix double-linking of drivers/usb/otg when ULPI is selected
To: linux-usb@vger.kernel.org, linux-omap@vger.kernel.org
Cc: beagleboard@googlegroups.com, daniel@caiaq.de, Bill Gatliff <bgat@billgatliff.com>
Message-ID: <1261151842-577-1-git-send-email-bgat@billgatliff.com>


This patch corrects a problem where drivers/usb/otg is linked twice
if CONFIG_USB_ULPI is selected, resulting in a build error (symbol
conflict). The files in that directory are properly linked already
as part of CONFIG_USB, and need not be indicated specifically for
CONFIG_USB_ULPI.

Signed-off-by: Bill Gatliff <bgat@billgatliff.com>
Acked-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Makefile |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/Makefile
+++ b/drivers/usb/Makefile
@@ -44,5 +44,3 @@ obj-y				+= early/
 
 obj-$(CONFIG_USB_ATM)		+= atm/
 obj-$(CONFIG_USB_SPEEDTOUCH)	+= atm/
-
-obj-$(CONFIG_USB_ULPI)		+= otg/
